projects/kubernetes/autoscaler/1-28/helm/patches/0002-Add-image-values.patch

```diff
@@ -1,14 +1,12 @@
-From 425c4836f440ea88974a969ba73028c56da366c8 Mon Sep 17 00:00:00 2001
+From ac5be4595d6db0af9a55c642f7e4fa24e7ed4513 Mon Sep 17 00:00:00 2001
 From: Prow Bot <prow@amazonaws.com>
-Date: Mon, 10 Jun 2024 21:25:54 +0000
+Date: Fri, 27 Sep 2024 18:36:19 -0700
 Subject: [PATCH] Add image values
```

Comment:
```suggestion
Subject: [PATCH 2/2] Add image values
```
Similar updates for every other patch in each release branch

Reply:
I believe the patch name itself is numbered and follows the same approach as it was handled in the previous commits.
The subject of a single patch is "[PATCH]" followed by the concatenation of lines from the commit message up to the first blank and when multiple patches are output, the subject prefix will instead be "[PATCH n/m] "

Reply:
https://github.com/aws/eks-anywhere-build-tooling/tree/main/projects/kubernetes/autoscaler/1-28/helm/patches
I see that the helm chart for 1-28 has multiple patches. Shouldn't the subject prefix follow the n/m format in this case?

Reply:
I believe it's more of a preference and doesn't impact how the patches are applied, here we were updating a single patch so it made more sense to generate this as a single patch which was followed in previous commits as well.